currencyConversion.playground/Contents.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+enum Currency {
+    case cad
+    case mxn
+}
+
+let usToCad: Double = 1.40
+let usToMxn: Double = 23.93
+
+var currency: Currency = .mxn
+
+
+
+func convert(dollars: Double) -> Double {
+    var dollars = dollars
+    switch currency {
+    case .cad:
+        dollars = dollars * usToCad
+    case .mxn:
+        dollars = dollars * usToMxn
+    }
+    return dollars
+}
+
+print(convert(dollars: 20))
+print(convert(dollars: 35))
+
+currency = .cad
+
+print(convert(dollars: 20))
+print(convert(dollars: 35))
+
+var currencyFormatter: NumberFormatter = {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .currencyAccounting
+    return formatter
+}()
+
+func convert(amountString: String) -> String? {
+    
+    let amount = Double(amountString)
+    
+    guard let Amount2 = amount
```

Comment:
Make sure to always have variables and constants start with a lowercase letter. In this instance. I would recommend preferring the same name (this seems to be the most popular convention), or prepending unwrapped or something similar, as it isn't actually a different amount, it is just the unwrapped value.

``` swift
guard let amount = amount
// or
guard let unwrappedAmount = amount
```